packages/hoppscotch-app/components/collections/ChooseType.vue

```diff
@@ -94,7 +95,20 @@ const emit = defineEmits<{
 const currentUser = useReadonlyStream(currentUserInfo$, null)
 
 const adapter = new TeamListAdapter(true)
-const myTeams = useReadonlyStream(adapter.teamList$, [])
+const myTeams = useReadonlyStream(adapter.teamList$, null)
+const REMEMBERED_TEAM_ID = useSetting("REMEMBERED_TEAM_ID")
+let teamListFetched = false
+
+watch(myTeams, (teams) => {
+  if (teams && !teamListFetched) {
+    teamListFetched = true
+    if (REMEMBERED_TEAM_ID.value && currentUser) {
+      selectedCollectionTab.value = "team-collections"
```

Comment:
This will again set the `Teams Collections` tab as active even if the user left `My Collections` selected. Better remove this line.
```suggestion
```